app/auth/auth_service.ts

```diff
@@ -44,6 +46,21 @@ export class AuthService {
           this.onUserRpcError(error);
         }
       });
+
+    setInterval(() => {
+      if (this.user) this.refreshToken();
+    }, TOKEN_REFRESH_INTERVAL_SEC * 1000);
```

Comment:
wdyt about defining the const in ms and skipping the conversion here? Since you're giving it a 30 min value anyway, cut out the middle man.

Reply:
done